controller.py

```diff
@@ -32,15 +33,29 @@ def add_type():
     database_connector.insert_type(database_connector.connect_to_db())
     return redirect("/administration")
 
-@app.route("/plans")
+@app.route("/plans", methods = ['POST', 'GET'])
 def render_plans_page():
-    list_of_periods = database_connector.fetch_periods(database_connector.connect_to_db())
-    list_of_categories = database_connector.fetch_categories(database_connector.connect_to_db())
-    return render_template("plans.html", category_list = list_of_categories, period_list = list_of_periods)
+    if request.method == 'GET':
+        list_of_periods = database_connector.fetch_periods(database_connector.connect_to_db())
+        period_id = list_of_periods[0][0]
+        list_of_plans = database_connector.fetch_plans(database_connector.connect_to_db(), period_id)
```

Comment:
ha period_id alapján keresed a planeket, akkor ezt mondja el a function neve is.